backend/controllers/metricsController.js

```diff
@@ -68,4 +68,29 @@ metricsController.getMemory = async () => {
   }
 };
 
+metricsController.getCPU = async () => {
+  // create query at current time
+  const currentDate = new Date().toISOString();
+  const query = `query_range?query=sum(rate(container_cpu_usage_seconds_total{image!=""}[2m])) by (pod)&start=${currentDate}&end=${currentDate}&step=1m`;
+  
+  try {
+    let cpuArr;
```

Comment:
I think we can remove line 77 and just add a `const` variable declaration on line 82.